src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -417,4 +429,32 @@ export class PiaIntakeService {
     if (!dto.submittedAt && dto.status === PiaIntakeStatusEnum.MPO_REVIEW)
       dto.submittedAt = new Date();
   };
+
+  /**
+   * @method validateJsonbFields
+   *
+   * @param userType - type of the logged in user
+   * It can be DRAFTER when initially creating the PIA or while editing it
+   * It shall also be DRAFTER when an MPO is drafting a PIA
+   * It can be MPO when MPO is reviewing someone else's PIA
+   *
+   * @description
+   * This method validates role access to the following fields, if needed:
+   * 1. collectionUseAndDisclosure
+   * 2. storingPersonalInformation
+   * 3. securityPersonalInformation
+   * 4. accuracyCorrectionAndRetention
+   * 5. personalInformationBanks
+   * 6. additionalRisks
+   */
```

Comment:
so this pr just validate the first tab ?

Reply:
clarified over the call. At the moment, we only have role checks in first tab.